File: Kata 6/Convert string to camel case/Convert string to camel case.cs
using System;
using System.Text.RegularExpressions;
using System.Linq;
public class Kata
{
  public static string ToCamelCase(string str)
  {
    return Regex.Replace(str, @"[_-](\w)", m => m.Groups[1].Value.ToUpper());
  }
  public static string ToCamelCase(string str)
  {   

    //return string.Join("", str.Split('_','-').Select((x,i) => i>0 ? x.Substring(0, 1).ToUpper() + x.Substring(1) : x));
	return string.Join("", str.Split('_','-').Select((x,i) => i==0 ? x:x.Substring(0, 1).ToUpper() + x.Substring(1)));
  }
       
}

File: Kata 6/Convert string to camel case/Convert string to camel case.py
import re
def to_camel_case(s):
    return s[0] + s.title().translate(None, "-_")[1:] if s else s
	
def to_camel_case(text):
    return ''.join(v.title() if i else v for i,v in enumerate(re.split('[_-]',text)))
	

def to_camel_case(text):
    a=[]
    for i,v in enumerate(re.split('[_-]',text)):
        if i:
            a.append(v.title())
        else:
            a.append(v)    
    
    return ''.join(a)